src/transform.js

```diff
@@ -42,6 +42,12 @@ export default class i18nTransform extends Transform {
     }
 
     this.options = { ...this.defaults, ...options }
+    this.options.i18nextOptions = {
+      ...options.i18nextOptions,
```

Comment:
Problem is that `pluralSeparator` option is also used in `helper.js`, so it would produce inconsistent behavior if `i18nextOptions.pluralSeparator` is set to another value than `pluralSeparator`. That's why I choose to keep the precedence on existing options (`namespaceSeparator` and `pluralSeparator`).

This reduce the risk of introducing a breaking change for existing users configurations too.